prov/efa/src/efa_av.c

```diff
@@ -438,6 +438,7 @@ struct efa_conn *efa_conn_alloc(struct efa_av *av, struct efa_ep_addr *raw_addr,
 		goto err_release;
 	}
 
+	memset(&reverse_av_entry->key, 0, sizeof(reverse_av_entry->key));
 	memcpy(&reverse_av_entry->key, &key, sizeof(key));
```

Comment:
This doesn't hurt anything, but is not necessary.  key is memset before it is initialized above and the memcpy overwrites reverse_av_entry->key completely.  [Note: I realize that I said this was needed.  That was a mistake.]